docs/en/gps_compass/rtk_gps.md

```diff
@@ -20,54 +20,59 @@ The RTK compatible devices below that are expected to work with PX4 (it omits di
-| [Navisys L1/L2 ZED-F9P RTK - Base only](https://www.navisys.com.tw/productdetail?name=GR901&class=RTK)    |                             F9P                             |          |                                  |                                                   |
-| [RaccoonLab L1/L2 ZED-F9P][RaccoonLab L1/L2 ZED-F9P]                                                      |                             F9P                             |  RM3100  |                ✓                 |                                                   |     |
-| [RaccoonLab L1/L2 ZED-F9P with external antenna][RaccnLabL1L2ZED-F9P ext_ant]                             |                             F9P                             |  RM3100  |                ✓                 |                                                   |
-| [Septentrio AsteRx-m3 Pro](../gps_compass/septentrio_asterx-rib.md)                                       |                           AsteRx                            |    ✓     |                                  |      [Septentrio Dual Antenna][SeptDualAnt]       |  ✓  |
-| [Septentrio mosaic-go](../gps_compass/septentrio_mosaic-go.md)                                            |                    mosaic X5 / mosaic H                     |    ✓     |                                  |      [Septentrio Dual Antenna][SeptDualAnt]       |  ✓  |
-| [SIRIUS RTK GNSS ROVER (F9P)](https://store-drotek.com/911-sirius-rtk-gnss-rover-f9p.html)                |                             F9P                             |    ✓     |                                  |                [Dual F9P][DualF9P]                |
-| [SparkFun GPS-RTK2 Board - ZED-F9P](https://www.sparkfun.com/products/15136)                              |                             F9P                             |    ✓     |                                  |                [Dual F9P][DualF9P]                |
-| [Trimble MB-Two](../gps_compass/rtk_gps_trimble_mb_two.md)                                                |                             F9P                             |    ✓     |                                  |                         ✓                         |     |
+| Device                                                                                                    |         GPS          | Compass  | [DroneCAN] |         [GPS Yaw]         | PPK |
+| :-------------------------------------------------------------------------------------------------------- | :------------------: | :------: | :--------: | :-----------------------: | :-: |
+| [ARK G5 RTK GPS](../dronecan/ark_g5_rtk_gps.md)                                                           |    [mosaic-G5 P3]    | IIS2MDC  |     ✓      |                           |     |
+| [ARK G5 RTK HEADING GPS](../dronecan/ark_g5_rtk_heading_gps.md)                                           |   [mosaic-G5 P3H]    | IIS2MDC  |     ✓      | [Heading][mosaic-G5 P3H]  |     |
```

Comment:
This just links to the product page - is this "Septentrio Dual Antenna" heading as linked from the other topics?